packages/auto-mcp-servers/src/bin/auto-experiences.ts

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js'
+// Import directly from the root index file
+import { autoExperiencesServer } from '../index.js'
+
```

Comment:
I'm not sure what was the initial design, but if the servers are supposed to be separate, we should import this like below. Otherwise the args of `autoExpereienceServer` will be combined with `auto-drive` MCP servers and will not run.
```suggestion
import { autoExperiencesServer } from '../auto-experiences/index.ts'
```

Reply:
For more context, I tried to run the auto-drive MCP and it was giving me error about missing `AGENT_PATH`.

Reply:
good catch